sparrow_mlpipes/bins/source.py

```diff
@@ -44,11 +44,14 @@ def make_source_bin(
         uridecodebin.connect("pad-added", on_pad_added, bin)
         uridecodebin.connect("child-added", on_child_added, bin)
     else:
-        multifilesrcbin = make_element(
-            "multifilesrc", "multi-file-source", location="%03d.jpeg", index=0
+        decodebin = make_element(
```

Comment:
Reading through various blogs I've seen:
`gst-launch-1.0 multifilesrc location="/foo/bar-%d.mp3" loop=true start-index=1 ! decodebin ! autoaudiosink`
So I thought I was save this element as "decodebin" vs. "uridecodebin" (referencing this site some: https://gstreamer.freedesktop.org/documentation/playback/uridecodebin.html?gi-language=python).

Regardless of what sort of decode bin I save it to, I get the following error:
`TypeError: <__gi__.GstMultiFileSrc object at 0x7f041a42ad00 (GstMultiFileSrc at 0x31540e0)>: unknown signal name: child-added`
So I think there's something different about a GstMultiFileSrc object that breaks this connect command.

Reply:
👀